openedx/core/djangoapps/bookmarks/tests/test_tasks.py

```diff
@@ -142,10 +145,11 @@ def test_calculate_course_xblocks_data(self, course_attr):
                     )
 
     @ddt.data(
-        ('course', 47),
-        ('other_course', 34)
+        ('course', 48),
+        ('other_course', 35)
     )
     @ddt.unpack
+    @skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Different query counts in CMS')
```

Comment:
@cpennington I figured it out-- when the test is run under LMS, there is a certificates signal handler listening to course publish signals. That ends up hitting the method that checks disabled_xblocks without the cached value already being present. I have updated the test to just increment the expected count by 1 (with a comment of course).